src/keys.rs

```diff
@@ -303,24 +309,21 @@ impl IssuanceAuthorizingKey {
 /// $\mathsf{ik}$ but stored here as a RedPallas verification key.
 ///
 /// [IssuanceZSA]: https://qed-it.github.io/zips/zip-0227#issuance-key-derivation
-#[derive(Debug, Clone, PartialOrd, Ord)]
-pub struct IssuanceValidatingKey(VerificationKey<IssuanceAuth>);
+#[derive(Debug, Clone)]
+pub struct IssuanceValidatingKey(schnorr::VerifyingKey);
 
 impl From<&IssuanceAuthorizingKey> for IssuanceValidatingKey {
     fn from(isk: &IssuanceAuthorizingKey) -> Self {
-        IssuanceValidatingKey((&(conditionally_negate(isk.derive_inner()))).into())
-    }
-}
-
-impl From<&IssuanceValidatingKey> for pallas::Point {
-    fn from(issuance_validating_key: &IssuanceValidatingKey) -> pallas::Point {
-        pallas::Point::from_bytes(&(&issuance_validating_key.0).into()).unwrap()
+        let schnorr_sk = schnorr::SigningKey::from_bytes(&isk.0).unwrap();
+        IssuanceValidatingKey(*schnorr_sk.verifying_key())
     }
 }
 
 impl PartialEq for IssuanceValidatingKey {
     fn eq(&self, other: &Self) -> bool {
-        <[u8; 32]>::from(&self.0).eq(&<[u8; 32]>::from(&other.0))
+        let temp1: [u8; 32] = self.to_bytes();
+        let temp2: [u8; 32] = other.to_bytes();
+        temp1.eq(&temp2)
```

Comment:
You could write it in one line
`self.to_bytes().eq(&other.to_bytes())`

Reply:
Done.